util/feeding.py

```diff
@@ -149,27 +134,24 @@ def split_audio_file(audio_path,
                      batch_size=1,
                      aggressiveness=3,
                      outlier_duration_ms=10000,
-                     outlier_batch_size=1):
-    sample_rate, _, sample_width = audio_format
-    multiplier = 1.0 / (1 << (8 * sample_width - 1))
-
+                     outlier_batch_size=1,
+                     exception_box=None):
```

Comment:
Why is the `multiplier` no longer needed?

Reply:
This is now done in a slightly different way as part of the audio conversion in `util.audio.pcm_to_np`: 
```
samples = samples.astype(np.float32) / np.iinfo(dtype).max
```